src/math/CompoundMath.sol

```diff
@@ -7,8 +7,8 @@ pragma solidity ^0.8.0;
 /// @dev Library to perform Compound's multiplication and division in an efficient way.
 library CompoundMath {
     /* CONSTANTS */
-
     // Only direct number constants and references to such constants are supported by inline assembly.
+
```

Comment:
Did you try, with an auto generated doc? I'm not certain it will work

Reply:
I tried and those comments do not show, but I think we do not want to show them, because the user should not care about how the implementation uses inline assembly.
That being said, I don't know how to have a comment that shows for a section, and not for a particular function/variable. Do you know how to do that @MerlinEgalite ?

Thinking about it, I'm not sure why the math constants are internal and not private in all our libraries. It's weird that the constant `MAX_UINT256` is defined both in `CompoundMath` and `PercentageMath`, no ? Wdyt @morpho-org/onchain-team ?

Reply:
because sometimes you would like to access the scaling factor such as `WAD`.

> It's weird that the constant MAX_UINT256 is defined both in CompoundMath and PercentageMath

What is you suggestion?

Reply:
> What is you suggestion?

I'll create a PR on top to show it.

What's your thoughts on the other questions @MerlinEgalite: showing or not showing the comment, how to show a comment for a whole section

Reply:
#112